docs/source/predict.rst

```diff
@@ -0,0 +1,102 @@
+dglke_predict: predicting entities/relations in a triplet
+-------------------------------------------
+The task is to predict missing entities or relations in a triplet. Blow shows an example that predicts top 5 most likely destination entities for every given source node and relation::
+
+   src  rel  dst   score
+    1    0    12   -5.11393
+    1    0    18   -6.10925
+    1    0    13   -6.66778
+    1    0    17   -6.81532
+    1    0    19   -6.83329
+    2    0    17   -5.09325
+    2    0    18   -5.42972
+    2    0    20   -5.61894
+    2    0    12   -5.75848
+    2    0    14   -5.94183
+
+Currently, it supports six models: TransE_l1, TransE_l2, RESCAL, DistMult, ComplEx, and RotatE.
+
+Four arguments are required to provide basic information for predicting missing entities or relations:
+
+  * ``--data_path``, The path containing the id mapping files, including both the entity ID mapping file and the relation ID mapping file. Default: ./data.
```

Comment:
the path only contains id mappings?

Reply:
The data_path is work with the config file.
In general if user use DGL-KE train the model, they do not need to specify the mapping files. They are all logged in the config file.

Reply:
I updated the help doc of --entity_mfile and --rel_mfile

Reply:
since the command can find the Id mappings from --entity_mfile, --rel_mfile and the config file, i don't feel it's necessary to provide --data_path. it's kind of redundant. it's also confusing, especially we have something called --data_files.